packages/core/test/keychain.spec.ts

```diff
@@ -0,0 +1,31 @@
+import "mocha";
+import { getDefaultLoggerOptions } from "@walletconnect/logger";
+import pino from "pino";
+
+import {
+  Core,
+  CORE_DEFAULT,
+  CORE_STORAGE_PREFIX,
+  KeyChain,
+  KEYCHAIN_CONTEXT,
+  KEYCHAIN_STORAGE_VERSION,
+} from "../src";
+import { expect, TEST_CORE_OPTIONS } from "./shared";
+import { ICore } from "@walletconnect/types";
+
+describe("Keychain", () => {
+  const logger = pino(getDefaultLoggerOptions({ level: CORE_DEFAULT.logger }));
+
+  let core: ICore;
+
+  beforeEach(() => {
```

Comment:
This looks like core can be instantiated on the same level as logger?

Reply:
Yeah definitely overkill here with the `beforeEach`, will simplify 👍 was copied across from another test